utils/utils.js

```diff
@@ -0,0 +1,35 @@
+export default{
+    toPattern(value, pattern){
```

Comment:
Essa é mais uma função que gostaria de entender melhor

Reply:
Também, talvez role deixa ela menor

Reply:
@PauloKlixto 

Cheguei nessa função depois de tentar de diversas maneiras transformar os padrões de data em um Regex. Mas para isso funcionar a função de data estava ficando muito complicada, com diversos replaces e diversos REGEX separados um do outro. Então resolvi fazer essa função onde você passa o valor e o padrão que quer ser aplicado no valor ex:

`08011991, 99/99/9999` =  `08/01/1991`

`87243945058, 999.999.999-99` =  `872.439.450-58`

na função vc também pode passar três tipos de padrões 

`A = só Letras
 9 = só Números
S = Letra e Número
` 
 Vou usar essa função também para quando for internacionalizar Telefone pq o padrão de telefone é muito absurdo.

@lucasyarid 

Essa já é a segunda versão dessa função já até dei uma diminuída nela , mas acho que ainda da para melhorar. Se tiver alguma ideia me fala que hoje ainda vou tentar diminuir

Reply:
É, acho que dá pra melhorar a função mas bora por partes, tá funcionando agora e você já tá entregando bastante valor nesse PR.

@spolidorio @PauloKlixto
Que que vocês acham de abrir um débito técnico e aí num tempo livre alguém tenta resolver?
É uma boa prática de algoritmo.

Reply:
Boa concordo